### PR TITLE
Complete flow of Design Thinking

### PR DESCRIPTION
**Pull request by arungupta007**

### CODE REVIEW

**Review by dev-manoranjan — COMMENTED** [review]
Please use svg images

File: src/screens/training/Debit.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      </View>
+      <View style={styles.body}>
+      <Text style={styles.heading}>{'Credit / Debit card'}</Text>
+      <View style={styles.spacer}  />
+      <View style={styles.debitCard}>
+        <View style={styles.debitHalf}>
+            <View style={{flex:2, backgroundColor:'#fff'}}>
+             <Image source={images.yesBank}/>
+            </View>
+            <View style={{flex: 8}}>
```
Comment (israrem) [inline src/screens/training/Debit.tsx:31]: Dont use inline styling

File: src/screens/training/Confirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <View style={styles.spacer}  />
+        <Text>{'Congratulations, your application has been sent'}</Text>
+      </View>
+      </View>
+      <View style={styles.bottom}>
+      <View style={styles.spacer}  />
+      <View style={styles.spacer}  />
+       <TouchableOpacity onPress={()=>{}}>
+        <View style={styles.buyButton}>
+        <Text style={{color:'#ffffff'}}>{'Buy Now'}</Text>
```
Comment (israrem) [inline src/screens/training/Confirmation.tsx:45]: use colors from styles/colors

**Comment by israrem** [comment]
image
Please check the cvv input 
and e-wallets container padding not matching with figma

File: src/screens/scholarships/AboutScholarship.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  SafeAreaView,
+  Text,
+  View,
+  FlatList,
+  ScrollView,
+  Alert
+} from 'react-native';
+import Button from '../../components/AppButton';
+import {styles}  from './styles';
+import Spacer from '../../components/Spacer';
```
Comment (israrem) [inline src/screens/scholarships/AboutScholarship.tsx:12]: use absolute paths

**Pull request merged**